cmd/jaeger/internal/extension/healthcheckextension/config.go

```diff
@@ -0,0 +1,71 @@
+package healthcheckextension
```

Comment:
why do we need to build our own health check extension? why can't we use upstream one (they have v1 and v2)

Reply:
Do you mean the [contrib's health check extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckextension)?
I tried directly configuring the yaml file to include the healthcheck extension, but unlike telemetery it dosen't work like that.
We have to create an extension like jaegerQuery, jaegerStorage and remoteSampling and then configure the yaml file to include that extension.
I got in touch with Pushkar, and asked him how he created the remoteSampling extension and he mentioned it was a derivative of contrib's remotesampling extension.
So, I was doing the same thing, creating an extension which is a derivative of their implementation.
Is there something else that needs to be done?

Reply:
I don't follow. Why should we build our own extension instead of using official one?

Reply:
By official one you are referring to the above link itself right?
So, are you implying that I should just import it as a package and use it?
Because, as per the roadmap for healthcheck I had in my mind, in order to configure the yaml file to include the healthcheck extension, the foremost step is to add the healthcheck code under the `extension` folder.

Reply:
I am referring to this one: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckv2extension

Yes, you need to configure it in YAML - it doesn't mean you need to implement it, just like we don't implement all other standard components from OTEL. You will need to include it in cmd/jaeger/internal/components.go

Reply:
Ohh okay, thanks a lot!
Looking into it right now.